management.md

```diff
@@ -35,5 +35,46 @@
 - What process will the team follow to find, report, and fix defects in the
   software project?
 
-- How will your team know when it is finished with the implementation of the
-  software project?
+  - Finding Defects:
+
+    In order to find the defects in a software project,
+    first check the error
+    messages terminal. Secondly, if there still are
```

Comment:
"error messages in terminal" would be a good way to clarify this point

Reply:
Thanks for the feedback! I fixed this issue in the latest commit.